src/example5.cpp

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: Intel Corporation
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <dr/mhp.hpp>
+#include <fmt/core.h>
+
+namespace mhp = dr::mhp;
+
+using T = float;
+using MDA = dr::mhp::distributed_mdarray<T, 2>;
+
+int main() {
+#ifdef SYCL_LANGUAGE_VERSION
```

Comment:
SYCL_LANGUAGE_VERSION is always defined, as we checked the presence of SYCL compiler

Reply:
Done